packages/devtools_app/lib/src/profiler/cpu_profile_transformer.dart

```diff
@@ -29,20 +96,9 @@ class CpuProfileTransformer {
         cpuProfileData.stackFrames[v[CpuProfileData.parentIdKey]],
         cpuProfileData,
       );
-    });
-    _setExclusiveSampleCounts(cpuProfileData);
-
-    cpuProfileData.processed = true;
-
-// TODO(kenz): investigate why this assert is firing again.
-// https://github.com/flutter/devtools/issues/1529.
-//    assert(
-//      cpuProfileData.profileMetaData.sampleCount ==
-//          cpuProfileData.cpuProfileRoot.inclusiveSampleCount,
-//      'SampleCount from response (${cpuProfileData.profileMetaData.sampleCount})'
-//      ' != sample count from root '
-//      '(${cpuProfileData.cpuProfileRoot.inclusiveSampleCount})',
-//    );
+    }
+    _stackFramesProcessed +=
```

Comment:
nit: this is correct but might as well just increment
_stackFramesProcessed within the for loop.
The performance hit won't matter and that way there can't be bugs if for example an exception is thrown processing a frame or if the logic is tweaked.